language-server/modules/langserver-core/src/test/resources/semantictokens/config/single-file/record.bal

```diff
@@ -0,0 +1,22 @@
+type User record {|
+    string name;
+|};
+
+type Payload record {|
+    record {|
+        *User;
+        string cardNo;
+    |} patient;
+|};
```

Comment:
Can you add a test for the open records as well. It will helps to check that the fix is applicable to both open and close records